pkg/virtctl/vm/vm.go

```diff
@@ -112,44 +110,21 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Cannot obtain KubeVirt client: %v", err)
 	}
 
-	options := k8smetav1.GetOptions{}
-	vm, err := virtClient.VirtualMachine(namespace).Get(vmiName, &options)
-	if err != nil {
-		return fmt.Errorf("Error fetching VirtualMachine: %v", err)
-	}
-
 	switch o.command {
-	case COMMAND_STOP, COMMAND_START:
-		running := false
-		if o.command == COMMAND_START {
-			running = true
+	case COMMAND_START:
+		err = virtClient.VirtualMachine(namespace).Start(vmiName)
+		if err != nil {
+			return fmt.Errorf("Error restarting VirtualMachine %v", err)
```

Comment:
i think the start and restart errors got switched around here.

this should be "Error starting..." and the Restart error should be "Error restarting..." but they're reversed. Probably a copy paste error.